src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,65 @@
+package step2
+
+/**
+ * 문자열 사칙 연산 계산기
+ * Created by Jaesungchi on 2022.05.05..
+ */
+
+fun main() {
+    val calculator = StringCalculator()
+    val input = readlnOrNull()
+    val output = calculator.startCalculate(input)
+    println(output)
+}
+
+class StringCalculator {
+    /**
+     * 문자열을 받고 계산을 시작하여 계산한값을 돌려준다.
+     */
+    fun startCalculate(input: String?): Int {
+        if (input.isNullOrBlank()) throw IllegalArgumentException(ErrorMessage.IS_NULL_OR_BLANK)
```

Comment:
아래와 같이 중괄호를 작성해주면 가독성이 더 좋아질 것 같아요!

```suggestion
        if (input.isNullOrBlank()) {
            throw IllegalArgumentException(ErrorMessage.IS_NULL_OR_BLANK)
        }
```